README.md

```diff
@@ -5,30 +5,35 @@ By performing a [GaAttachTx](https://aeternity.com/protocol/generalized_accounts
 
 ## Features & limitations
 
-- Provide a list of co-signers
-    - minimum 1
+- Provide a list of signers
+    - minimum 2
 - Provide the amount of confirmations need for a tx to be authorized
     - minimum 2
-    - must not exceed amount of co-signers
+    - must not exceed amount of signers
 - It's only possible to propose one tx at the same time & in order to be able to propose a new tx the current tx needs to be:
     - authorized (executed)
     - expired
     - revoked
-- Co-signers & amount of required confirmations cannot be changed afterwards
+- Signers & amount of required confirmations cannot be changed after attaching the GA MultiSig
+- Currently the contract needs to handle fee protection to prevent malicious behavior of miners
+    - reasonable default values have been defined by default
+    - the values can be changed by any signer
+    - it's expected that a future hardfork introduces handling the fee protection on protocol level
+        - to prepare for this the fee protection handling on contract level can be disabled later on
 
 ## Contract entrypoints
 
 ### Stateful
 - `init(int, list(address))`
     - params
         - `int` is the amount of confirmations required for a tx to be authorized
-        - `list(address)` a list of co-signers which will be included in addition to the address of the account which is converted into a GA
+        - `list(address)` a list of signers which will be included in addition to the address of the account which is converted into a GA
```

Comment:
The address of the account which is converted is not useful as a signer since it can never sign anything 😅